src/Symfony/Component/Finder/Iterator/FilterIterator.php

```diff
@@ -12,9 +12,9 @@
 namespace Symfony\Component\Finder\Iterator;
 
 /**
- * This iterator just overrides the rewind method in order to correct a PHP bug.
+ * This iterator just overrides the rewind method in order to correct a PHP bug, which existed before version 5.5.23.
```

Comment:
I double-checked before making the change. It's not fixed in 5.5.22. Also, the fix is listed only in
http://php.net/ChangeLog-5.php#5.5.23